README.md

```diff
@@ -25,44 +25,43 @@ Small and Medium business accounting and ERP
  6. [License](#license)
 
 
-# DESCRIPTION
+# Description
 
-LedgerSMB is a free web-based double-entry accounting system, featuring
-quotation, ordering, invoicing, projects, timecards, inventory management,
-shipping and more ...
+LedgerSMB is a free and interactive web-based double-entry accounting system, featuring
+quotations, ordering, invoicing, projects, timecards, inventory management,
+shipping, and more...
 
 Directly send orders and invoices from the built-in e-mail function to your
-customers or RFQs (request for quotation) to your vendors with PDF attachments,
-from anywhere in the world with the browser-based UI.
+customers or RFQs (Request For Quotation) to your vendors, complete with PDF attachments,
+from anywhere in the world with an easy-to-use browser-based UI.
 
-With its data stored in the enterprise-strength `PostgreSQL` open source
+With its data stored in the enterprise-strength `PostgreSQL` open source relational
 database system, the system is known to operate smoothly for businesses with
 thousands of transactions per week.
 
-Customer visible output is fully customizable in templates, allowing easy and
-fast customization. Supported output formats are PDF, CSV, HTML, ODF and more.
+Customer-visible output is fully customizable in templates, allowing easy and
+fast customization.  Supported output formats include PDF, CSV, HTML, ODF, and more.
```

Comment:
Double space after sentences is no longer the norm.

Reply:
Two spaces after the end of sentences is still common in many parts of the world, such as in Canada (where I am).  I also see it commonly used in professional documents, such as reports and documentation.  I also see two spaces after sentences used in other languages too, including Chinese.  I see this in past and modern works, so it appears to continue to be "the norm" from my perspective.

Perhaps it's a nation-specific standard that I'm not encountering as often since I probably read more Canadian content?

Reply:
Two spaces were the norm when you and I were young and using typewriters. No longer the case. I'm in Québec.